src/basic/time-util.c

```diff
@@ -883,16 +883,16 @@ typedef struct ParseTimestampResult {
 } ParseTimestampResult;
 
 int parse_timestamp(const char *t, usec_t *usec) {
-        char *last_space, *timezone = NULL;
+        char *last_space, *t_timezone = NULL;
```

Comment:
So this is really a bug in clang. gcc had the same issue that they'd stupidly warn about local variables shadowing globals, but they fixed it a while back. I'd prefer to improve clang rather then pile workarounds in the code, but since this is just a local variable name, let's merge this.

Reply:
hmm, while it's not really an error to shadow things in C, and llvm is wrong with making this an error, I think it's probably a good idea to avoid the shadowing anyway, but I am not sure `t_timezone` is the best choice... Let's maybe just call this local variable "tz" or so...